src/status_im/ui/screens/multiaccounts/views.cljs

```diff
@@ -36,15 +35,13 @@
                                                        :height 19}}]])
                   :chevron   true}])
 
+(defn topbar-button []
```

Comment:
How is this `topbar-button` used?

Reply:
`(.registerNavigationButtonPressedListener
   (.events Navigation)`
called from here